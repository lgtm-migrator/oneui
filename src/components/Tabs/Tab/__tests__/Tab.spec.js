import '@testing-library/jest-dom/extend-expect';
import React from 'react';
import { render } from '@testing-library/react';
import Tab from '../Tab';

describe('<Tab> that renders tab content', () => {
    it('should render a tab', () => {
        const { container } = render(
            <Tab label="My tab" id="myTab">
                Some content
            </Tab>
        );
        expect(container).toMatchSnapshot();
    });
});
