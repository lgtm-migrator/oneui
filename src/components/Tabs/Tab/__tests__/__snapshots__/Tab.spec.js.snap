// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tab> that renders tab content should render a tab 1`] = `
<div>
  Some content
</div>
`;
